aries_vcx/tests/test_mysql_wallet.rs

```diff
@@ -0,0 +1,102 @@
+#[macro_use]
+extern crate log;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+#[cfg(feature = "mysql_test")]
+#[macro_use]
+extern crate sqlx;
+
+#[cfg(feature = "mysql_test")]
+mod dbtests {
+    use sqlx::{Connection, MySqlConnection};
+    use sqlx::Executor;
+    use sqlx::prelude;
+    use sqlx::prelude::*;
+
+    #[cfg(feature = "mysql_test")]
+    pub async fn setup_mysql_walletdb() -> Result<String, sqlx::Error> {
```

Comment:
Elsewhere, we established a pattern where functions like this are put into a `test_utils` module (with a feature of the same name, which is [included](https://github.com/hyperledger/aries-vcx/pull/345/files#diff-3e6ca8175bf22b164caebe6ae1ce69a105be029e5c329cb15cfe15c260f7e73cR22) under `mysql_test` feature anyway)